infinitest-intellij/src/main/java/org/infinitest/intellij/idea/IdeaModuleSettings.java

```diff
@@ -88,14 +89,26 @@ public RuntimeEnvironment getRuntimeEnvironment() {
 	 * 
 	 * @return A list of all of the output directories for the project
 	 */
-	private List<File> listOutputDirectories() {
-		List<File> outputDirectories = new ArrayList<File>();
+    private List<File> listOutputDirectories() {
+        List<File> outputDirectories = new ArrayList<File>();
+
+        addOutputDirectory(outputDirectories, CompilerPaths.getModuleOutputPath(module, false));
```

Comment:
I added constants to have a clearer understanding of what the second parameter means.  Even though it's only a boolean, it's worth it.